src/engine/OptionalJoin.h

```diff
@@ -4,13 +4,9 @@
 //         Florian Kramer (florian.kramer@netpun.uni-freiburg.de)
 #pragma once
 
-#include <list>
-
 #include "./Operation.h"
 #include "./QueryExecutionTree.h"
```

Comment:
While you are at it, you can unify the format of the remaining includes in the files you touch here:)